### ISSUE TITLE
ttkDesigner:  Adding a layout to a newWindow, and saving it corrupts save file

### ISSUE THREAD

**Issue by castben** [ceccopierangiolieugenio/pyTermTk#136]
Hi I just another crash,  I've created a new window and put a grid layout on it (I did not changed property Layout from window itself).

Added few widgets and save it, then when I tried to open it I got:
``` 
Traceback (most recent call last):
  File "/home/larry/.local/bin/ttkDesigner", line 8, in <module>
    sys.exit(main())
  File "/home/larry/.local/lib/python3.10/site-packages/ttkDesigner/app/main.py", line 47, in main
    root.mainloop()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 163, in mainloop
    self._mainLoop()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 172, in _mainLoop
    self._input.start()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/TTkTerm/input.py", line 77, in start
    self.key_process(stdinRead)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/TTkTerm/input.py", line 161, in key_process
    self.inputEvent.emit(kevt, mevt)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/signal.py", line 140, in emit
    slot(*args[sl], **kwargs)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 180, in _processInput
    self._mouse_event(mevt)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/ttk.py", line 210, in _mouse_event
    focusWidget.mouseEvent(nmevt)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkWidgets/widget.py", line 448, in mouseEvent
    if self.mouseReleaseEvent(evt):
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkWidgets/button.py", line 233, in mouseReleaseEvent
    self.clicked.emit()
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/signal.py", line 140, in emit
    slot(*args[sl], **kwargs)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkWidgets/TTkPickers/filepicker.py", line 294, in _open
    self.pathPicked.emit(fileName)
  File "/home/larry/.local/lib/python3.10/site-packages/TermTk/TTkCore/signal.py", line 140, in emit
    slot(*args[sl], **kwargs)
  File "/home/larry/.local/lib/python3.10/site-packages/ttkDesigner/app/designer.py", line 252, in _openFile
    sw = SuperWidget.loadDict(self, self._windowEditor.viewport(), dd['tui'])
  File "/home/larry/.local/lib/python3.10/site-packages/ttkDesigner/app/superobj/superwidget.py", line 169, in loadDict
    sch._superLayout.setDropBorder(0)
AttributeError: 'SuperLayoutGrid' object has no attribute '_superLayout'. Did you mean: 'setLayout'?
```
Now I think I know why, because I put a layout on top of window default layout (which I shouldn't).

I realised it after reading your tutorial on TTK designer. (Yeah I know I should see that first) 

May I also sugest to prevent to do this and save it like that? because it seems it corrupts save file.

I did test again, this time avoiding to add a layout and modifying directly the window layout setting and it worked, I saved my form, closed program, re-opened and load my form back and this time it worked.

This example is `corrupted` and what I did was:

1) Create a new window
2) add a layout (any layout)
3) Save it
4) close program
5) Open program
6) Open saved file
then it crashed.

This is the saved tui  file:
```
{
 "version": "1.0.0",
 "tui": {
  "class": "TTkWindow",
  "params": {
   "Name": "MainWindow",
   "Position": [
    4,
    2
   ],
   "Size": [
    115,
    38
   ],
   "Min Width": 2,
   "Min Height": 4,
   "Max Width": 65536,
   "Max Height": 65536,
   "Padding": [
    3,
    1,
    1,
    1
   ],
   "Layout": "TTkLayout",
   "Visible": true,
   "Enabled": true,
   "ToolTip": "\u001b[0m",
   "Border": true,
   "Title": "\u001b[0m",
   "Window Flags": 134217728
  },
  "layout": {
   "class": "TTkLayout",
   "params": {
    "Geometry": [
     0,
     0,
     113,
     34
    ]
   },
   "children": [
    {
     "class": "TTkLayout",
     "params": {
      "Geometry": [
       4,
       4,
       30,
       10
      ]
     },
     "children": [],
     "row": 0,
     "col": 0,
     "rowspan": 1,
     "colspan": 1
    }
   ]
  }
 },
 "connections": []
}
```

**Comment by ceccopierangiolieugenio** [comment]
You should be allowed to add a layout on top of another layout, I need to investigate it

**Comment by ceccopierangiolieugenio** [comment]
Fixed in ttkDesigner **0.30.0a50**
You can try to load this example: [Nested.Layouts.tui.json.zip](https://github.com/ceccopierangiolieugenio/pyTermTk/files/11407256/Nested.Layouts.tui.json.zip)
